docs/example-scenario/apps/ecommerce-scenario.md

```diff
@@ -4,7 +4,7 @@ description: Proven scenario of hosting an e-commerce site on Azure
 author: masonch
 ms.date: 7/13/18
 ---
-# E-Commerce front-end on Azure
+# E-Commerce frontend on Azure
```

Comment:
Should be two words ("front end") when used as a noun.

Reply:
This one I'm not sure about, front-end is definitely wrong as that would be the way you write a compound adjective. frontend/backend are the more likely to be hit on via search, and in terms of a technology stack frontend/backend make sense as a thing in themselves.  

I think we need to standadize because some of the other articles talk about backends as well.